content/en/docs/setup/install/multiple-controlplanes/index.md

```diff
@@ -0,0 +1,299 @@
+      name: "usergroup-2-peerauth"
+      namespace: "usergroup-2"
+    spec:
+      mtls:
+        mode: STRICT
+    EOF
+    {{< /text >}}
+
+{{< warning >}}
+In a fully scoped environment where each control plane is associated with its resources through proper namespace labeling, there is no need for the default webhook
+configurations provided by Istio. However it is the responsibility of the Mesh Administrator to make sure no requests bypass the validation step.
+{{< /warning >}}
+
+The below output shows `istiod-default-validator` and `istio-revision-tag-default-usergroup-1`, which are the default webhook configuration available in Istio deployment used for handling requests coming from resources which are not labeled with any revision. It would be ideal to remove them from the charts for the current use case, as they are primarily used to handle revision based upgrade. `istio-revision-tag-default-tenant-1` handles the sidecar injection for pods that are labeled with `istio-injection=enabled`, which is not needed for cases where we are adding usergroup specific labels.
+
+    {{< text bash >}}
+    $ kubectl get validatingwebhookconfiguration
+    NAME                                      WEBHOOKS   AGE
+    istio-validator-usergroup-1-usergroup-1   1          18m
+    istio-validator-usergroup-2-usergroup-2   1          18m
+    istiod-default-validator                  1          18m
+    {{< /text >}}
+
+    {{< text bash >}}
+    $ kubectl get mutatingwebhookconfiguration
+    NAME                                             WEBHOOKS   AGE
+    istio-revision-tag-default-usergroup-1           4          18m
+    istio-sidecar-injector-usergroup-1-usergroup-1   2          19m
+    istio-sidecar-injector-usergroup-2-usergroup-2   2          18m
+    {{< /text >}}
+
+### Verify Multiple Control Plane Creation
+
+1. Check the labels on the system namespaces per user group
+
+    {{< text bash >}}
+    $ kubectl get ns usergroup-1 usergroup2 --show-labels
+    NAME              STATUS   AGE     LABELS
+    usergroup-1       Active   13m     kubernetes.io/metadata.name=usergroup-1,usergroup=usergroup-1
+    usergroup-2       Active   12m     kubernetes.io/metadata.name=usergroup-2,usergroup=usergroup-2
+    {{< /text >}}
+
+1. Verify the namespaces in which the control planes are deployed
+
+    {{< text bash >}}
+    $ kubectl get pods -n usergroup-1
+    NAMESPACE     NAME                                     READY   STATUS    RESTARTS         AGE
+    usergroup-1   istiod-usergroup-1-5ccc849b5f-wnqd6      1/1     Running   0                12m
+    {{< /text >}}
+
+    {{< text bash >}}
+    $ kubectl get pods -n usergroup-2
+    NAMESPACE     NAME                                     READY   STATUS    RESTARTS         AGE
+    usergroup-2   istiod-usergroup-2-658d6458f7-slpd9      1/1     Running   0                12m
+    {{< /text >}}
+
+    You will notice that one istiod deployment per usergroup is created in the specified namespaces.
```

Comment:
```suggestion
    You will notice that one istiod deployment per usergroup is created in the specified namespaces.

1. Run the following commands to list the installed webhooks:

    {{< text bash >}}
    $ kubectl get validatingwebhookconfiguration
    NAME                                      WEBHOOKS   AGE
    istio-validator-usergroup-1-usergroup-1   1          18m
    istio-validator-usergroup-2-usergroup-2   1          18m
    istiod-default-validator                  1          18m
    {{< /text >}}

    {{< text bash >}}
    $ kubectl get mutatingwebhookconfiguration
    NAME                                             WEBHOOKS   AGE
    istio-revision-tag-default-usergroup-1           4          18m
    istio-sidecar-injector-usergroup-1-usergroup-1   2          19m
    istio-sidecar-injector-usergroup-2-usergroup-2   2          18m
    {{< /text >}}

    Note that the output includes `istiod-default-validator` and `istio-revision-tag-default-usergroup-1`, which are the default webhook configurations used for handling requests coming from resources which are not associated with any revision. In a fully scoped environment where every control plane is associated with its resources through proper namespace labeling, there is no need for these default webhook configurations. They will never be invoked.
```

Reply:
This is taken care, but still shows up as not resolved, not sure why.